README.md

```diff
@@ -72,14 +72,17 @@ If the command you are trying to use is not compatible, `pip_api` will raise a
   > * `Distribution.editable` (`bool`): Whether the distribution is editable or not
   > Optionally takes a `local` parameter to filter out globally-installed packages
 
-* `pip_api.parse_requirements(filename, options=None, include_invalid=False)`
-  > Takes a path to a filename of a Requirements file. Returns a mapping from package name to a [`packaging.requirements.Requirement`](https://packaging.pypa.io/en/latest/requirements/#packaging.requirements.Requirement) object with the following attributes:
+* `pip_api.parse_requirements(filename, options=None, include_invalid=False, strict_hashes=False)`
+  > Takes a path to a filename of a Requirements file. Returns a mapping from package name to a [`packaging.requirements.Requirement`](https://packaging.pypa.io/en/latest/requirements/#packaging.requirements.Requirement) subclass object with the following attributes:
```

Comment:
```suggestion
  > Takes a path to a filename of a Requirements file. Returns a mapping from package name to a [`pip_api.Requirement`] object (subclass of [`packaging.requirements.Requirement`](https://packaging.pypa.io/en/latest/requirements/#packaging.requirements.Requirement)) with the following attributes:
```

We probably need to move this class into a public API somewhere.

Reply:
Yeah, agreed. We currently export it as `pip_api.Requirement`, so I can change the docs to that.